benchmark/README.md

```diff
@@ -186,3 +202,21 @@ public func run_YourTestName(N: Int) {
 
 The current set of tags are defined by the `BenchmarkCategory` enum in
 `TestsUtils.swift` .
+
+Testing the Benchmark Drivers
+-----------------------------
+When working on tests, after the initial build
+````
+swift-source$ ./swift/utils/build-script -R -B
+````
+you can rebuild just the benchmarks:
+````
+swift-source$ export SWIFT_BUILD_DIR=`pwd`/build/Ninja-ReleaseAssert/swift-macosx-x86_64
+swift-source$ ninja -C ${SWIFT_BUILD_DIR} swift-benchmark-macosx-x86_64
```

Comment:
You don't need to export SWIFT_BUILD_DIR.

Reply:
It illustrates what it should be, because it isn't obvious, as you start learning about Swift's build process. I had to learn it the hard way, so I've included it in documentation. You'll rewrite this whole file soon enough, so it ultimately doesn't matter. Take it as feedback for what to include in the rewrite.